native/cocos/platform/android/java/src/com/cocos/lib/CocosActivity.java

```diff
@@ -110,100 +85,53 @@ private void setImmersiveMode() {
         }
     }
 
+    //Deprecated, for compatibility, keep this interface for now
     protected void initView() {
-        mRootLayout = findViewById(contentViewId);
-        if (mWebViewHelper == null) {
-            mWebViewHelper = new CocosWebViewHelper(mRootLayout);
-        }
-
-        if (mVideoHelper == null) {
-            mVideoHelper = new CocosVideoHelper(this, mRootLayout);
-        }
+        mCocosEngine.initView(findViewById(contentViewId));
```

Comment:
Why do we need `CocosEngine.init` and `CocosEngine.initView` both?
Should `init` do all the initialization work ?